starter-packs/refinery-deployment/cloudformation-aws/image-builder/aws.pkr.hcl

```diff
@@ -71,10 +71,16 @@ build {
     destination = "/tmp/honeycomb-metrics-config.yaml"
   }
 
+  provisioner "file" {
+    source = "configure-refinery.sh"
+    destination = "/tmp/configure-refinery.sh"
+  }
+
   provisioner "shell" {
     environment_vars = [
-      "REFINERY_RELEASE=1.16.0",
-      "OTEL_CONFIG_RELEASE=1.4.0"
+      "REFINERY_RELEASE=1.18.0",
```

Comment:
Wonder if we need to pin the version like so - would it maybe be better supplied as an env car?

Reply:
The reason for pinning is because in the AWS Marketplace world, the shipped AMI is the artifact you "sell".  So this hopes to put some control around it. 

there are variables in Packer and we could conceivably follow [this pattern](https://github.com/aws/amazon-ecs-ami/blob/main/variables.pkr.hcl) like the AWS ECS image pipelines.   But that felt like a lot more complexity and less obvious for contributors so I wasn't sure if now was the right time in the project to do so.